apps/explorer/src/hooks/useMediaQuery.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { useEffect, useState } from 'react';
+
+/**
+ * values taken from tailwind.config.js
+ */
+export const BREAK_POINTS = {
+    sm: 640,
+    md: 768,
+    lg: 1024,
+    xl: 1280,
+    '2xl': 1536,
+};
+
+export function useMediaQuery(query: string) {
+    const [matches, setMatches] = useState(false);
+
+    useEffect(() => {
+        const media = window.matchMedia(query);
+        if (media.matches !== matches) {
```

Comment:
Just set it no matter what, React can optimize the re-render. This also means you don't need to depend on `matches`